### PR TITLE
Add some more ARIA

### PR DESCRIPTION
**Pull request by zorkow**
Adds some more aria. In particular 
* fixes the missing aria-checked elements
* adds separator role and orientation
* menu roles
* close button role and proper label attribute.
The last might clash with #1262

### CODE REVIEW

File: unpacked/extensions/MathMenu.js [hunk 1]
```diff
@@ -598,11 +598,13 @@
...
          ondragstart: FALSE, onselectstart: FALSE, onselectend: FALSE,
          ontouchstart: MENU.Touchstart, ontouchend: MENU.Touchend,
-         className: "MathJax_MenuItem", menuItem: this},
+         className: "MathJax_MenuItem", role: this.role,
+         menuItem: this},
         def);
     },
 
     Create: function (menu) {
+      console.log('create');
```
Comment (dpvc) [inline unpacked/extensions/MathMenu.js]: This log message will need to be removed.

File: unpacked/extensions/MathMenu.js [hunk 2]
```diff
@@ -898,19 +909,24 @@
...
           var item = child[i].menuItem;
-          if (item && item.variable === this.variable)
-            {child[i].firstChild.style.display = "none"}
+          if (item && item.variable === this.variable) {
+            child[i].firstChild.style.display = "none";
+          }
         }
         menu.firstChild.display = "";
         CONFIG.settings[this.variable] = this.value;
+        console.log('Changing');
```
Comment (dpvc) [inline unpacked/extensions/MathMenu.js]: This log message will need to be removed.

**Comment by dpvc** [comment]
I think you are right about the conflicts with the localization in #1262.  We should merge this one first (once the `console.log()` calls are removed), I'll fix the `more-menu` branch, and then we can merge that.

**Comment by zorkow** [comment]
Oopss.... forgot to push the cleaned up version... sorry.

**Pull request merged**